packages/core/src/generators/qwik/component.ts

```diff
@@ -82,7 +84,10 @@ export function addComponent(
   opts: { isRoot?: boolean; shareStyles?: boolean } = {},
 ) {
   const _opts = { isRoot: false, shareStyles: false, ...opts };
-  compileAwayBuilderComponentsFromTree(component, compileAwayComponents);
+  compileAwayBuilderComponentsFromTree(
+    component,
+    omit(compileAwayComponents, ['Image']),
+  );
```

Comment:
I am slightly weary of pulling in `lodash` dependencies given how notorious it is for increasing bundle size.

For a relatively simple use-case like this one, how do you feel about omitting using the spread operator?

```suggestion
  const { Image, ...compileAwayComponentsWithoutImage } = compileAwayComponents
  compileAwayBuilderComponentsFromTree(
    component,
    compileAwayComponentsWithoutImage,
  );
```